create-security-certificates.md

```diff
@@ -1,103 +1,199 @@
 
-~~~ shell
-$ cockroach cert create-ca \
---ca-cert=certs/ca.cert \
---ca-key=certs/ca.key 
-~~~
+1. 	Create a safe directory for the CA key:
 
-### Create the certificate and key for a node
+	~~~ shell
+	$ mkdir my-safe-directory
+	~~~
 
-~~~ shell
-$ cockroach cert create-node \
-node1.example.com \
-node1.another-example.com \
---ca-cert=certs/ca.cert \
---ca-key=certs/ca.key \
---cert=certs/node.cert \
---key=certs/node.key
-~~~
+2. 	Generate the CA certificate and key:
+
+	~~~ shell
+	$ cockroach cert create-ca \
+	--certs-dir=certs \
+	--ca-key=my-safe-directory/ca.key
+
+	$ ls -l certs
+	~~~
+
+	~~~
+	-rw-r--r--  1 maxroach  maxroach  1.1K Apr 19 09:35 ca.crt
+	~~~
+
+### Create the certificate and key pairs for nodes
+
+1. 	Create the certificate and key for the first node:
+
+	~~~ shell
+	$ cockroach cert create-node \
+	node1.example.com \
+	node1.another-example.com \
+	--certs-dir=certs \
+	--ca-key=my-safe-directory/ca.key
 
-### Create the certificate and key for a client
+	$ ls -l certs
+	~~~
+
+	~~~
+	-rw-r--r--  1 maxroach  maxroach  1.1K Apr 19 09:35 ca.crt
+	-rw-r--r--  1 maxroach  maxroach  1.2K Apr 19 09:36 node.crt
+	-rw-------  1 maxroach  maxroach  1.6K Apr 19 09:36 node.key
+	~~~
+
+2. 	Upload the files to the first node.
+
+3. 	Create the certificate and key for the second node, using the `--overwrite` flag to replace the files created for the first node:
+
+	~~~ shell
+	$ cockroach cert create-node \
+	node2.example.com \
+	node2.another-example.com \
+	--certs-dir=certs \
+	--ca-key=my-safe-directory/ca.key \
+	--overwrite
+
+	$ ls -l certs
+	~~~
+
+	~~~
+	-rw-r--r--  1 maxroach  maxroach  1.1K Apr 19 09:35 ca.crt
+	-rw-r--r--  1 maxroach  maxroach  1.2K Apr 19 09:40 node.crt
+	-rw-------  1 maxroach  maxroach  1.6K Apr 19 09:40 node.key
+	~~~
+
+4. 	Upload the files to the second node.
+
+5. 	Repeat for each additional node.
+
+### Create the certificate and key pair for a client
 
 ~~~ shell
 $ cockroach cert create-client \
 maxroach \
---ca-cert=certs/ca.cert \
---ca-key=certs/ca.key \
---cert=certs/maxroach.cert \
---key=certs/maxroach.key
+--certs-dir=certs \
+--ca-key=my-safe-directory/ca.key
+
+$ ls -l certs
+~~~
+
+~~~
+-rw-r--r--  1 maxroach  maxroach  1.1K Apr 19 09:35 ca.crt
+-rw-r--r--  1 maxroach  maxroach  1.1K Apr 19 09:39 client.maxroach.crt
+-rw-------  1 maxroach  maxroach  1.6K Apr 19 09:39 client.maxroach.key
+-rw-r--r--  1 maxroach  maxroach  1.2K Apr 19 09:36 node.crt
+-rw-------  1 maxroach  maxroach  1.6K Apr 19 09:36 node.key
+~~~
+
+### List certificates and keys
+
+~~~ shell
+$ cockroach cert list \
+--certs-dir=certs
+~~~
+
+~~~
+Certificate directory: certs
++-----------------------+---------------------+---------------------+---------------+
+|         Usage         |  Certificate File   |      Key File       |     Notes     |
++-----------------------+---------------------+---------------------+---------------+
+| Certificate Authority | ca.crt              |                     |
```

Comment:
ok, so I messed up the code instead. Fixed in https://github.com/cockroachdb/cockroach/pull/15198
We can probably fix the border in the docs even though this won't be in the beta. Up to you though.